operator/src/main/java/org/keycloak/operator/controllers/KeycloakRealmImportJob.java

```diff
@@ -165,7 +165,7 @@ private void buildKeycloakJobContainer(Container keycloakContainer) {
         var runBuild = (keycloak.getSpec().getImage() == null) ? "/opt/keycloak/bin/kc.sh build && " : "";
 
         var commandArgs = List.of("-c",
-                runBuild + "/opt/keycloak/bin/kc.sh import --file='" + importMntPath + getRealmName() + "-realm.json' " + override);
+                runBuild + "/opt/keycloak/bin/kc.sh import --optimized --file='" + importMntPath + getRealmName() + "-realm.json' " + override);
```

Comment:
Why not to leverage the auto-build functionality here?

Reply:
There is the logic above that the build is not run when there is a custom built image defined, and I didn't want to change that part.